content/en/building/reference/app-settings/transitions.md

```diff
@@ -824,3 +824,47 @@ Configuration is validated when Sentinel starts. Issues with the configuration w
 Errors occurring during the client-side transition will be recorded in the browser's console. This is where problems with processing reports from the replace forms will be logged. 
 
 Errors occurring during the server-side transition will be recorded in the Sentinel logs and on the contact doc for the original user. So, if the client-side transition marks the original user for replacement, but Sentinel fails to create the new user, the failure will be recorded on the original contact doc in the `errors` array.
+
+### Resetting the sentinel sequence ID to skip a large backlog
+
+In some cases, a CHT instance may develop a very large sentinel backlog that would take too long to process, preventing new transitions from being applied to incoming reports. In these situations, you can reset the sentinel sequence ID to skip processing the backlog.
+
+Follow these steps to reset the sentinel sequence ID:
+
+1. **Confirm you have a sentinel backlog** in watchdog. This will likely be in the thousands or millions. While it may be decreasing, you'll notice that it could take weeks or months to reach zero. During this backlog processing, no new sentinel transitions will be processed:
+
+   {{< figure src="sentinel-backlog.png" link="sentinel-backlog.png" alt="Screenshot showing a sentinel backlog in watchdog" >}}
+
+2. **Get the current sequence ID** for the medic database via:
+
+   ```
+   curl -qs https://medic:password@your-instance-url:port/medic/ | jq ".update_seq"
+   ```
```

Comment:
``````suggestion
2. **Get the current sequence ID** for the medic database via a `curl` call. Be sure to replace `<password>` and `<cht-url>` with the correct values:

   ```
   curl -qs https://medic:<password>@<cht-url>/medic/ | jq ".update_seq"
   ```
``````